R/verbatim_popup.R

```diff
@@ -61,7 +57,9 @@ verbatim_popup_ui <- function(id, button_label, type = c("button", "link"), ...)
 #' @param style (`logical(1)`) whether to style the `verbatim_content` using `styler::style_text`.
 #' If `verbatim_content` is a `condition` or `reactive` holding `condition` then this argument is ignored
 #' @param disabled (`reactive(1)`) the `shiny` reactive value holding a `logical`. The popup button is disabled
-#' when the flag is `TRUE` and enabled otherwise
+#' when the flag is `TRUE` and enabled otherwise.
+#' Please take into account that `shinyjs` is the dependency for this functionality so the `shinyjs::useShinyjs()` call
```

Comment:
Can we just include useShinyjs() in the UI of this element. It's a singleton so multiple calls shouldn't make a difference.

Reply:
@Polkas what about this? Would you agree to include `shinyjs::useShinyjs` so we don't need to notify anyone.

Reply:
the package has the shinyjs dependency so not including it will NOT reduce the dependency for sb who want to use this functionality.
Then I agree with you that adding `shinyjs::useShinyjs` it is a good move.